src/snowflake/snowpark/functions.py

```diff
@@ -5931,6 +5931,24 @@ def vector_inner_product(v1: ColumnOrName, v2: ColumnOrName) -> Column:
     return builtin("vector_inner_product")(v1, v2)
 
 
+def ln(c: ColumnOrLiteral) -> Column:
+    """Returns the natrual log product of given column expression
+    Example::
```

Comment:
```suggestion
    """Returns the natrual log of given column expression
    Example::
```
I don't think we should call it product. see the official description: https://docs.snowflake.com/en/sql-reference/functions/ln